tests/scripts/components-psasim.sh

```diff
@@ -10,37 +10,34 @@
 ################################################################
 
 # Helper function for controlling (start & stop) the psasim server.
-helper_psasim_server () {
+helper_psasim_server() {
     OPERATION=$1
     if [ "$OPERATION" == "start" ]; then
-    (
-        cd tests
         msg "start server in tests"
-        psa-client-server/psasim/test/start_server.sh
+        (
+            cd tests
+            ../$PSASIM_PATH/test/start_server.sh
+        )
         msg "start server in tf-psa-crypto/tests"
-        cd ../tf-psa-crypto/tests
-        ../../tests/psa-client-server/psasim/test/start_server.sh
-    )
+        (
+            cd tf-psa-crypto/tests
+            ../../$PSASIM_PATH/test/start_server.sh
+        )
     else
-    (
-        msg "terminate servers and cleanup"
-        tests/psa-client-server/psasim//test/kill_servers.sh
-
-        # Remove temporary files and logs
-        cd tests
-        rm -f psa_notify_*
-        rm -f psa_service_*
-        rm -f psa_server.log
-
-        cd ../tf-psa-crypto/tests
-        rm -f psa_notify_*
-        rm -f psa_service_*
-        rm -f psa_server.log
-    )
+        msg "terminate server in tests"
+        (
+            cd tests
+            ../$PSASIM_PATH/test/kill_servers.sh
+        )
+        msg "terminate server in tf-psa-crypto/tests"
+        (
+            cd tf-psa-crypto/tests
+            ../../$PSASIM_PATH/test/kill_servers.sh
```

Comment:
Presumably the only point of the second `kill_servers.sh` is to remove the temporary files, since the first one will have killed the actual processes and deleted all the IPCS queues

Reply:
Yes, indeed that's why I tried to rename this bash file as `kill_server.sh` on some former version of this PR. However even in that case the name is not correct because as you pointed out the first call kills _all_ the server instances and the 2nd call only removes temporary files... If you prefer I can create ad hoc bash files with proper names for these 2 different tasks so it will be clearer which one does what.

Reply:
Rather than more extensive changes, how about just adding comments so that people reading this file know we are aware what's going on? Something like this?

```
        msg "terminate server in tests"
        (
            cd tests
            # This will kill both servers and clean up all the message queues,
            # and clear temporary files in tests
            ../$PSASIM_PATH/test/kill_servers.sh
        )
        msg "terminate server in tf-psa-crypto/tests"
        (
            cd tf-psa-crypto/tests
            # This just clears temporary files in tf-psa-crypto/tests
            ../../$PSASIM_PATH/test/kill_servers.sh
        )
```

Reply:
(please feel free to improve the comments!)

Reply:
Actually they looked OK to me so I simply copy/paste them. Since the commit history is extremely short for this PR and the changes introduced here are minimal, I chose to update the already existing commit instead of adding a new one.